docs/src/data/tailwind-migration-status.yaml

```diff
@@ -88,7 +88,7 @@ StopoverArrow: false
 Switch: true
 Tab: false
 TabPanel: false
-Table: false
+Table: true
 TableBody: false
```

Comment:
Please update all the sub-components as well.

Reply:
right 🤦‍♀️ didn't notice those